src/core/node_descriptors.rs

```diff
@@ -187,8 +194,7 @@ impl NodeDescriptor for CommitOidDescriptor {
         _glyphs: &Glyphs,
         object: &NodeObject,
     ) -> eyre::Result<Option<StyledString>> {
-        let oid = object.get_oid();
-        let oid = &oid.to_string()[..8];
```

Comment:
Is removing `[..8]` ok?

Reply:
Let's move the `[..8]` case to `NodeObject::get_short_id` in the `GarbageCollected` case. In that case, it doesn't really matter what OID we show, since the object is no longer available anyways, so we might as well keep it short to make it easy to read the output.